docs/pages/web/selectlist.js

```diff
@@ -194,6 +253,7 @@ export default function SelectListPage({ generatedDocGen }: {| generatedDocGen:
     };
 
     return (
+<<<<<<< HEAD:docs/pages/web/selectlist.js
```

Comment:
Need to clean this @dangerismycat

Reply:
Ah thanks, good catch! More encouragement to keep converting pages to the new code example style 😬